rabbit-escape-engine/src/rabbitescape/engine/behaviours/Walking.java

```diff
@@ -274,6 +274,7 @@ public boolean behave( World world, Rabbit rabbit, State state )
                 return true;
             }
             case RABBIT_TURNING_LEFT_TO_RIGHT:
+                rabbit.onSlope = false; // Intentional fall-through
             case RABBIT_TURNING_LEFT_TO_RIGHT_RISING:
```

Comment:
Seems like onSlope is not well-defined enough if we are having to do random fixups like this... Maybe onSlope should become false at the beginning of each time step.